pom.xml

```diff
@@ -1477,6 +1495,7 @@
                     <artifactId>scalatest-maven-plugin</artifactId>
                     <version>${maven.plugin.scalatest.version}</version>
                     <configuration>
+                        <argLine>-Xmx4g -Xss4m -XX:ReservedCodeCacheSize=${codeCacheSize} ${extraJavaTestArgs} -Dio.netty.tryReflectionSetAccessible=true</argLine>
```

Comment:
Define `argLine` property rather than in configuration, otherwise you will have some troubles integrating other plugin (i.e jacoco).